ui_curses.c

```diff
@@ -2329,7 +2333,9 @@ static void init_curses(void)
 static void notify_init(void)
 {
 	int fildes[2];
-	pipe(fildes);
+	if (pipe(fildes) == -1) {
+		d_print("pipe failed: %s\n", strerror(errno));
```

Comment:
Sure, what's the right logging function to print an error message?

Do you think the other d_print() calls are fine, or should they also lead to cmus exiting?